types/logger/tokenbucket.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package logger
+
+import (
+	"time"
+)
+
+type tokenBucket struct {
```

Comment:
can you document this briefly, at least to say what how it differs from https://pkg.go.dev/golang.org/x/time/rate#Limiter ?

Reply:
added.